### PR TITLE
[BE] Fix dependabot security errors

### PR DESCRIPTION
**Pull request by ZainRizvi**
Fixes https://github.com/pytorch/pytorch/security/dependabot/36 and https://github.com/pytorch/pytorch/security/dependabot/37 by deleting spurious dependency

### CODE REVIEW

**Comment by huydhn** [comment]
Is this package available on conda now?  For the context, @malfet upgraded it in https://github.com/pytorch/pytorch/pull/126805, but https://github.com/pytorch/pytorch/pull/126989 downgrade it again because the package is not yet on conda.  A comment on https://github.com/pytorch/pytorch/pull/126989#issuecomment-2127986574 is to use pip instead?

**Comment by Smartappli** [comment]
> Is this package available on conda now? For the context, @malfet upgraded it in #126805, but #126989 downgrade it again because the package is not yet on conda. A comment on [#126989 (comment)](https://github.com/pytorch/pytorch/pull/126989#issuecomment-2127986574) is to use pip instead?

https://anaconda.org/conda-forge/requests

**Comment by huydhn** [comment]
IIRC, we don't use conda-forge in CI by default

**Comment by malfet** [comment]
I wonder, what is the purpose of those workflows to begin with. And why do they need requests?

File: .github/requirements/conda-env-iOS.txt [hunk 1]
```diff
@@ -3,6 +3,6 @@ cmake=3.22.1
 ninja=1.10.2
 numpy=1.23.3
 pyyaml=6.0
-requests=2.31.0
+requests=2.32.0
```
Comment (malfet) [inline .github/requirements/conda-env-iOS.txt:6]:
```suggestion
```

File: .github/requirements/conda-env-Linux-X64.txt [hunk 2]
```diff
@@ -4,6 +4,6 @@ mkl-include=2022.1.0
 ninja=1.10.2
 numpy=1.23.3
 pyyaml=6.0
-requests=2.31.0
+requests=2.32.0
```
Comment (malfet) [inline .github/requirements/conda-env-Linux-X64.txt:7]:
```suggestion
```

**Review by malfet — COMMENTED** [review]
Let's delete those and see what will happen

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**